backend/src/main/java/de/neuefische/teddyscoronadiaries/service/WeatherService.java

```diff
@@ -0,0 +1,65 @@
+package de.neuefische.teddyscoronadiaries.service;
+
+import de.neuefische.teddyscoronadiaries.db.ProvinceMongoDb;
+import de.neuefische.teddyscoronadiaries.metaweatherapi.model.WeatherData;
+import de.neuefische.teddyscoronadiaries.metaweatherapi.service.MetaWeatherApiService;
+import de.neuefische.teddyscoronadiaries.model.province.ProvinceData;
+import de.neuefische.teddyscoronadiaries.model.weather.ProvinceCapitalWeatherData;
+import de.neuefische.teddyscoronadiaries.model.weather.WeatherCategories;
+import de.neuefische.teddyscoronadiaries.utils.CurrentDateUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class WeatherService {
+
+    private final MetaWeatherApiService metaWeatherApiService;
+    private final ProvinceMongoDb provinceMongoDb;
```

Comment:
`interface PagingAndSorting` already hides your database implementation, so there is no need to reflect the specific implementation within the namespace outside of the interface: `CategoryMongoDb -> CategoryDb`